be/src/column/const_column.cpp

```diff
@@ -34,6 +34,10 @@ void ConstColumn::append_value_multiple_times(const Column& src, uint32_t index,
     append(src, index, size);
 }
 
+Status ConstColumn::update_rows(const Column& src, const uint32_t* indexes) {
+    return _data->update_rows(src, indexes);
```

Comment:
NotSupported

Reply:
> NotSupported

done, fix in https://github.com/StarRocks/starrocks/pull/2870/commits/02fed55409ca6b210b66225164c3741596038a12